clients/docs/ccloud.rst

```diff
@@ -0,0 +1,229 @@
+----------------------------------------
+
+.. include:: includes/client-example-schema-registry-3.rst
+
+
+Produce Avro Records
+~~~~~~~~~~~~~~~~~~~~
+
+.. Should steps 1 through 3 below be under this  "Produce Avro Records" section or before it?
+
+#. As described in the `Confluent Cloud
+   quickstart
+   <https://docs.confluent.io/current/quickstart/cloud-quickstart/schema-registry.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud>`__,
+   in the |ccloud| GUI, enable |ccloud| |sr| and create an API key and
+   secret to connect to it.
+
+#. Create the topic in |ccloud|.
+
+   .. code-block:: bash
+
+      ccloud kafka topic create test2
+
+#. Create a file that has the schema of your message payload.
+
+   .. code-block:: bash
+
+      echo '{"type":"record","name":"myrecord","fields":[{"name":"count","type":"int"}]}' > schema.json
+
+.. should the user verify anything after running the following
+   command or is there specific output they should see after running the command?
+
+#. Run the :ref:`Confluent Cloud CLI producer <ccloud_kafka_topic_produce>`
+   writing messages to topic ``test2``, passing in arguments for:
+
+   -  ``--value-format avro``: use Avro data format for the value part of the
+      message
+   -  ``--schema``: the path to the schema file
+   -  ``--parse-key --delimiter ,``: pass key and value, separated by a comma
+
+   .. code-block:: bash
+
+      ccloud kafka topic produce test2 --value-format avro --schema schema.json --parse-key --delimiter ,
+
+      .. note::
+
+          The first time you run this command, you must provide user credentials
+          for |ccloud| |sr|.
+
+.. Is there a page where we can direct users to for help in providing their credentials for Confluent Cloud Schema Registry?
+
+#. Type a few messages, using a ``,`` as the separator between the message key and value:
+
+   .. code-block:: bash
+
+      alice,{"count":3}
+      alice,{"count":4}
+      alice,{"count":5}
+
+#. When you are done, press ``Ctrl-C``.
+
+.. Not sure the correct source code link for this one. I only saw two .sh files in the ccloud directory, but we have
+   basic consumer/producer and then Avro consumer/producer. So assuming we'll need 4 source code links in total for this page?
+
+
+Consume Avro Records
+~~~~~~~~~~~~~~~~~~~~
+
+#. Run the :ref:`Confluent Confluent CLI consumer <ccloud_kafka_topic_consume>`
+   reading messages from topic ``test2``, passing in arguments for:
+
+   -  ``-b``: print all messages from the beginning of the topic
+   -  ``--value-format avro``: use Avro data format for the value part of the
+      message
+   -  ``--print-key``: print key and value (by default, it only prints value)
+
+   .. code-block:: bash
+
+      ccloud kafka topic consume test2 -b --value-format avro --print-key
+
+
+You should see the messages you typed in the previous step.
+.. "previous step" is referring to step 3 in the "Produce Records" section?
+
```

Comment:
@ybyzek  Just to confirm is "previous step" above referring to step 3 in the "Produce Records" section?

Reply:
Right, previous step being the section in `Produce Avro Records`

Reply:
cool beans